docs-src/docs/dataset-exploration/prediction-format.md

```diff
@@ -16,25 +16,36 @@ The current API for uploading predictions supports the following geometries:
 | Bounding box           | `bbox`          | Bounding box in 2D             |
 | Bitmaps (segmentation) | `image`         | Segmentation bitmap for images |
 
-Note that all geometries should be specified under frames rather than in the root of the pre-annotation. The rotation of
-cuboids should be the same as that in [exports](../openlabel/openlabel-format.md#rotation-of-cuboids). 2D geometries
-should be expressed in pixel coordinates. See [coordinate systems](../kognic-io/coordinate_systems.md) for more
-information.
+The rotation of cuboids should be the same as that in [exports](../openlabel/openlabel-format.md#rotation-of-cuboids) (see [coordinate systems](../kognic-io/coordinate_systems.md) for more information). 2D geometries should be expressed in pixel coordinates.
 
-`frame_properties` is required, same as for uploading pre annotations. For non-video
-data, `frame_properties.external_id` will be resolved automatically if it is left as an empty string.
-`frame_properties.timestamp` will be ignored for non-video data and can therefore be set to 0. `frame_properties.stream`
-can be left as an empty dict.
+For this API, the relevant parts (keys) are `frames`, `objects`, `streams`, `ontologies` and `metadata`. The last one (`metadata`) is the easiet one, and should just read `schema_version": "1.0.0"` (see examples below for full context). Also `stream` is straightforward, and should specify what sensors (cameras, lidars, ...) there are and what their name, like `sensor_name: {"type": "camera"}` or `sensor_name: {"type": "lidar"}`. Again, see the examples below for full context.
 
-For cuboids and bounding boxes, existence confidence can be provided by specifying an attribute called `confidence`. It is not required and will be set to 1.0 if it is left empty. If provided, it must be defined as a numeric value between 0.0 and 1.0. Existence confidence is set to 0.85 in the examples below. The `object_data.type` will show up as the class name in the tool.
+All parts of a prediction that is time-varying throughout a sequence is described in `frames`, such as corodinates and dynamic properties. Each frame in the sequence is represented by a key-value pair under `frames`. The key is the `frame_id`, and should look like
+```json
+frame_id: {
```

Comment:
I don't think this `frame_id` is every used but I am not 100% sure

Reply:
No it's not. And I don't think we claim it is? But it's still the frame_id in the openlabel, right?